addon/store.js

```diff
@@ -131,13 +123,41 @@ var Store = Ember.Object.extend({
 
             var attr = params[0];
             var value = options[attr];
-            var func = function(m) {
+
+            return this._findWithFilterFunc(type, function filterFunction(m) {
                 return get(m, attr) === value;
-            };
-            return this._findWithFilterFunc(type, func);
+            });
         }
+
         return this._findByIdComputed(type, options);
     },
+    findOne(type) {
+        var store = this;
+
+        return Ember.ObjectProxy.extend({
```

Comment:
looks like you already achieved this as you mentioned (model is using the getOwner api now). We might need to do a version bump later this week (still never published beta.7 w/ the unsubscribe api as I didn't want to add this and then remove it)